src/theme/book.js

```diff
@@ -455,6 +455,26 @@ function playpen_text(playpen) {
         }
     }, { passive: true });
 
+    document.addEventListener('keydown', function (e) {
+        if (e.altKey || e.ctrlKey || e.metaKey || e.shiftKey) { return; }
+
+        switch (e.key) {
+            case 'F9':
+                if (html.classList.contains("sidebar-hidden")) {
```

Comment:
Could you extract this into a separate function (since it is the same logic as in the `click` handler)? We could then just call it from the switch and the `click` handler would be

```js
sidebarToggleButton.addEventListener('click', toggleSidebar);
```

Reply:
Did.